src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -129,3 +129,5 @@ own style.  The link below shows the example output from the ``documentation_gen
    :maxdepth: 1
 
    example1_expected_documentation
+   mfem_coefficient_expected_documentation
```

Comment:
We should move/add a new page on documentation generation in a separate page.  It took me a bit to find these files

Reply:
Not sure if this needs to be in the quickstart other than mentioning it. Follow up PR maybe?

Reply:
Will be addressed in #500